src/app/pages/audit/components/audit/audit.component.html

```diff
@@ -1,4 +1,12 @@
 <ix-page-header>
+  @if (isHaLicensed()) {
+    <p>{{ 'Controller' | translate }}</p>
+    <mat-button-toggle-group [value]="controllerType()" (change)="controllerTypeChanged($event)">
```

Comment:
there is similar `ix-button-group` that accepts options$

Reply:
@denysbutenko  we may add a new ticket to convert all of these to use `<ix-button-group />` then.

<img width="353" alt="Screenshot 2024-09-26 at 13 44 35" src="https://github.com/user-attachments/assets/3a8b846e-4f0b-40f4-82cd-63745fed224f">